apps/api/views_drf.py

```diff
@@ -474,3 +477,24 @@ def delete(self, request, addon_id, version_id):
         version = self.get_object()
         version.delete()
         return Response(status=204)
+
+
+class GithubView(APIView):
+
+    def post(self, request):
+        if request.META.get('HTTP_X_GITHUB_EVENT') != 'pull_request':
+            # That's ok, we are just going to ignore it, we'll return a 2xx
+            # response so github doesn't report it as an error.
+            return Response({}, status=202)
```

Comment:
The definition of 202 at https://httpstatuses.com/202 is "The request might or might not eventually be acted upon, as it might be disallowed when processing actually takes place." which seemed ok. I'm not fussed though and github doesn't seem to care :)